app/sidekiq/lighthouse/poll_form526_pdf.rb

```diff
@@ -0,0 +1,110 @@
+# frozen_string_literal: true
+
+require 'lighthouse/benefits_claims/service'
+require 'sentry_logging'
+require 'logging/third_party_transaction'
+require 'sidekiq/form526_job_status_tracker/job_tracker'
+require 'sidekiq/form526_job_status_tracker/metrics'
+
+module Lighthouse
+  class PollForm526Pdf
+    include Sidekiq::Job
+    include Sidekiq::Form526JobStatusTracker::JobTracker
+    extend Logging::ThirdPartyTransaction::MethodWrapper
+    extend ActiveSupport::Concern
+
+    attr_accessor :submission_id
+
+    STATSD_KEY_PREFIX = 'worker.lighthouse.poll_form526_pdf'
```

Comment:
Where is `STATSD_KEY_PREFIX` used?

Reply:
@freeheeling confirmed locally that it's behaving as expected! thanks for the callout